pandas/core/series.py

```diff
@@ -4396,6 +4396,20 @@ def aggregate(self, func=None, axis=0, *args, **kwargs):
 
     agg = aggregate
 
+    @deprecate_nonkeyword_arguments(version=None, allowed_args=["self"])
+    @doc(NDFrame.any, **_shared_doc_kwargs)
+    def any(
+        self: Series,
+        axis: Axis = 0,
+        bool_only: bool | None = None,
+        skipna: bool = True,
+        level: Level | None = None,
+        **kwargs,
+    ) -> Series | bool:
+        return super().any(
+            axis=axis, bool_only=bool_only, skipna=skipna, level=level, **kwargs
+        )
```

Comment:
The issue is that this path isn't actually hit - this is because `any` is defined in 

That's because if you run `df.any(0)`, this code gets called:

https://github.com/pandas-dev/pandas/blob/79fb2debb14e77d6d4af9c4db058e6f994507e29/pandas/core/generic.py#L10928-L10948

, not the functions you've added in `frame.py` and `series.py`. That's why the test is failing.